rest/src/main/java/com/netflix/conductor/rest/controllers/TaskResource.java

```diff
@@ -103,8 +104,11 @@ public void log(@PathVariable("taskId") String taskId, @RequestBody String log)
 
     @GetMapping("/{taskId}/log")
     @Operation(summary = "Get Task Execution Logs")
-    public List<TaskExecLog> getTaskLogs(@PathVariable("taskId") String taskId) {
-        return taskService.getTaskLogs(taskId);
+    public ResponseEntity<List<TaskExecLog>> getTaskLogs(@PathVariable("taskId") String taskId) {
```

Comment:
consider using https://github.com/conductor-oss/conductor/blob/386dde51069bff2280efc1b1cbb572b6da81a6da/core/src/main/java/com/netflix/conductor/core/exception/NotFoundException.java when the entity is not found.

Reply:
The code has been updated to utilize the `NotFoundException` class in all the relevant places where an entity is not found. Thank you for your input!
<img width="1383" alt="image" src="https://github.com/conductor-oss/conductor/assets/47558150/ddd2fc30-38ab-4f0b-aa50-8de9575fc978">
<img width="1403" alt="image" src="https://github.com/conductor-oss/conductor/assets/47558150/adec208a-6e85-448f-b35d-c5b17c920cd9">
<img width="1409" alt="image" src="https://github.com/conductor-oss/conductor/assets/47558150/db3539f3-890f-4e41-b3b8-e6fe94bfe853">
<img width="1397" alt="image" src="https://github.com/conductor-oss/conductor/assets/47558150/179f3946-917e-4549-910e-40b90f420c29">